armi/operators/snapshots.py

```diff
@@ -90,3 +87,8 @@ def setStateToDefault(cs):
         from armi.operators.runTypes import RunTypes
 
         return cs.modified(newSettings={"runType": RunTypes.STANDARD})
+
+    @property
+    def atEOL(self):
+        """NOTE: This operator's atEOL method behaves very differently than other operators."""
```

Comment:
you could add some context that snapshots don't really have an EOL since they can go in any order and may or may not contain the last time node form the load database. It kind of sets the context.